.github/workflows/all-checks-passed.yml

```diff
@@ -0,0 +1,26 @@
+name: all-checks-passed
+on:
+  pull_request:
+    types:
+      [
+        opened,
```

Comment:
Shouldn't we include `unlabeled` event as well?

Reply:
good point - make sense to add it if we use `labeled`. Applied in [808a3ca](https://github.com/kyma-project/infrastructure-manager/pull/668/commits/808a3ca17ea851a75e83d5020e2d5dd128c71697)